hack/multikueue-e2e-test.sh

```diff
@@ -60,62 +65,30 @@ function startup {
         fi
 
-        cluster_create "$MANAGER_KIND_CLUSTER_NAME" "${ARTIFACTS}/manager-cluster.kind.yaml"
-        cluster_create "$WORKER1_KIND_CLUSTER_NAME" "$SOURCE_DIR/multikueue/worker-cluster.kind.yaml"
-        cluster_create "$WORKER2_KIND_CLUSTER_NAME" "$SOURCE_DIR/multikueue/worker-cluster.kind.yaml"
-    fi
-}
-
-function kind_load {
-    prepare_docker_images
-
-    if [ "$CREATE_KIND_CLUSTER" == 'true' ]
-    then
-        cluster_kind_load "$MANAGER_KIND_CLUSTER_NAME"
-        cluster_kind_load "$WORKER1_KIND_CLUSTER_NAME"
-        cluster_kind_load "$WORKER2_KIND_CLUSTER_NAME"
+        cluster_create "$MANAGER_KIND_CLUSTER_NAME" "${ARTIFACTS}/manager-cluster.kind.yaml" "$MANAGER_KUBECONFIG" &
+        cluster_create "$WORKER1_KIND_CLUSTER_NAME" "$SOURCE_DIR/multikueue/worker-cluster.kind.yaml" "$WORKER1_KUBECONFIG" &
+        cluster_create "$WORKER2_KIND_CLUSTER_NAME" "$SOURCE_DIR/multikueue/worker-cluster.kind.yaml" "$WORKER2_KUBECONFIG" &
     fi
-
-    # JOBSET SETUP
-    install_jobset "$MANAGER_KIND_CLUSTER_NAME"
-    install_jobset "$WORKER1_KIND_CLUSTER_NAME"
-    install_jobset "$WORKER2_KIND_CLUSTER_NAME"
-
-    # APPWRAPPER SETUP
-    install_appwrapper "$MANAGER_KIND_CLUSTER_NAME"
-    install_appwrapper "$WORKER1_KIND_CLUSTER_NAME"
-    install_appwrapper "$WORKER2_KIND_CLUSTER_NAME"
-
-    # KUBEFLOW SETUP
-    # In order for MPI-operator and Training-operator to work on the same cluster it is required that:
-    # 1. 'kubeflow.org_mpijobs.yaml' is removed from base/crds/kustomization.yaml - https://github.com/kubeflow/training-operator/issues/1930
-    # 2. Training-operator deployment is modified to enable all kubeflow jobs except for mpi -  https://github.com/kubeflow/training-operator/issues/1777
-    install_kubeflow "$MANAGER_KIND_CLUSTER_NAME"
-    install_kubeflow "$WORKER1_KIND_CLUSTER_NAME"
-    install_kubeflow "$WORKER2_KIND_CLUSTER_NAME"
-    
-    ## MPI
-    install_mpi "$MANAGER_KIND_CLUSTER_NAME"
-    install_mpi "$WORKER1_KIND_CLUSTER_NAME"
-    install_mpi "$WORKER2_KIND_CLUSTER_NAME"
-
-    ## KUBERAY
-    install_kuberay "$MANAGER_KIND_CLUSTER_NAME"
-    install_kuberay "$WORKER1_KIND_CLUSTER_NAME"
-    install_kuberay "$WORKER2_KIND_CLUSTER_NAME"
 }
 
 function kueue_deploy {
-    (cd config/components/manager && $KUSTOMIZE edit set image controller="$IMAGE_TAG")
+    (cd "${ROOT_DIR}/config/components/manager" && $KUSTOMIZE edit set image controller="$IMAGE_TAG")
 
-    cluster_kueue_deploy "$MANAGER_KIND_CLUSTER_NAME"
-    cluster_kueue_deploy "$WORKER1_KIND_CLUSTER_NAME"
-    cluster_kueue_deploy "$WORKER2_KIND_CLUSTER_NAME"
+    cluster_kueue_deploy "$MANAGER_KUBECONFIG"
+    cluster_kueue_deploy "$WORKER1_KUBECONFIG"
+    cluster_kueue_deploy "$WORKER2_KUBECONFIG"
 }
 
 trap cleanup EXIT
-startup
-kind_load
+startup 
+prepare_docker_images
+wait # for clusters creation
+( kind_load "$MANAGER_KIND_CLUSTER_NAME" "$MANAGER_KUBECONFIG" ) &
```

Comment:
```suggestion
kind_load "$MANAGER_KIND_CLUSTER_NAME" "$MANAGER_KUBECONFIG" &
```
nit: just for consistency